### PR TITLE
Add ApexF7

### PR DESCRIPTION
**Pull request by x4FF3**
Hi,

pls add ApexF7 FC. Board documentation is prepared, PR for BF repo will follow shortly.

KR
David

### CODE REVIEW

File: configs/default/APEX-APEXF7.config [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Betaflight / STM32F7X2 (S7X2) 4.1.6 Apr 25 2020 / 05:12:26 (283bda8bf) MSP API: 1.42
+# manufacturer_id: APEX   board_name: APEXF7   custom defaults: NO
```
Comment (mikeller) [inline configs/default/APEX-APEXF7.config:2]: Please remove this line.

Comment (x4FF3) [reply configs/default/APEX-APEXF7.config]: done :)

Comment (mikeller) [reply configs/default/APEX-APEXF7.config]: Now it is missing the `# Betaflight...` line - this one needs to stay in, the `# manufacturer_id` line needs to go.

Comment (x4FF3) [reply configs/default/APEX-APEXF7.config]: sorry, just realized that.

**Pull request merged**